src/harness/fake_sas.py

```diff
@@ -176,7 +176,15 @@ def _GetSuccessResponse(self):
   def _GetMissingParamResponse(self):
     return {'responseCode': MISSING_PARAM}
 
-
+  def InjectEscZone(self, request, ssl_cert=None, ssl_key=None)
+    zoneData = req['zoneDate']
+    if(zoneData['usage'] == 'EXCLUSION_ZONE'):
+      idAttributes = zoneData['id'].split('/')
```

Comment:
Why can't we just return zoneData['id'] directly? Fake SAS isn't supposed to be very smart, it just needs to return things which are syntactically correct.